src/components/project/ProjectTasks.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+function TaskTable(props) {
+  return (
+    <div className='tab-pane active'>
+      <table className='table'>
```

Comment:
We have universal component Table.js in components/global folder. Could you please use it to avoid code duplication?